server/common/src/main/java/org/eclipse/lsp/cobol/common/dialects/DialectProcessingContext.java

```diff
@@ -14,20 +14,19 @@
  */
 package org.eclipse.lsp.cobol.common.dialects;
 
+import java.util.ArrayList;
+import java.util.List;
 import lombok.Builder;
 import lombok.Value;
-import org.eclipse.lsp.cobol.common.copybook.CopybookConfig;
+import org.eclipse.lsp.cobol.common.AnalysisConfig;
 import org.eclipse.lsp.cobol.common.mapping.ExtendedDocument;
 import org.eclipse.lsp.cobol.common.model.tree.Node;
 
-import java.util.ArrayList;
-import java.util.List;
-
 /** Contains all needed data for dialect processing */
 @Value
 @Builder(toBuilder = true)
 public class DialectProcessingContext {
-  CopybookConfig copybookConfig;
+  AnalysisConfig config;
```

Comment:
Updated this model in common module. 
** This can be a breaking change for implementer
Reason -  A dialect can depend on the COBOL LS config (for e.g. compiler directive or a dialect config or a cobol config)